src/applications/mhv/medical-records/api/MrApi.js

```diff
@@ -145,19 +143,9 @@ export const getAllergy = id => {
  * Get a patient's vaccines
  * @returns list of patient's vaccines in FHIR format
  */
-export const getVaccineList = runningUnitTest => {
-  if (
-    (environment.BUILDTYPE === 'localhost' && IS_TESTING) ||
-    runningUnitTest
-  ) {
-    return apiRequest(`${apiBasePath}/medical_records/vaccines`, {
-      headers,
-    });
-  }
-  return new Promise(resolve => {
-    setTimeout(() => {
-      resolve(vaccines);
-    }, 1000);
+export const getVaccineList = () => {
+  return apiRequest(`${apiBasePath}/medical_records/vaccines`, {
+    headers,
   });
 };
```

Comment:
I am in favor making the Vaccine APIs use real data, but were you and Matt able to access vaccine data from the backend? I just tried it locally and it prints your error. I want to make sure it works before we push this change to staging.

I looked into it and the error I'm seeing is in the vaccines reducer. It's not checking if the response is empty before trying to grab records.
```
ERROR! TypeError: Cannot read properties of undefined (reading 'map')
    at vaccineReducer (vaccines.js:58:12)
```